packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1187,7 +1187,7 @@ class Align extends SingleChildRenderObjectWidget {
   /// edge of the parent. Other values interpolate (and extrapolate) linearly.
```

Comment:
I'd like to see unit tests for the widgets to which you added the ability to use FractionalOffsetGeometry, including some cases of weird combinations, to prove out whether the FractionalOffsetGeometry logic makes as much sense as the EdgeInsetsGeometry logic.

Reply:
(e.g. testing edge cases of changing from topLeft to topStart and seeing if it works right, or testing lerping, or whatnot)

Reply:
Added.